autoware_launch/config/perception/object_recognition/tracking/multi_object_tracker/multi_object_tracker_node.param.yaml

```diff
@@ -0,0 +1,18 @@
+/**:
+  ros__parameters:
+    # default tracker models for each class
+    car_tracker: "multi_vehicle_tracker"
+    truck_tracker: "multi_vehicle_tracker"
+    bus_tracker: "multi_vehicle_tracker"
+    trailer_tracker: "multi_vehicle_tracker"
+    pedestrian_tracker: "pedestrian_and_bicycle_tracker"
+    bicycle_tracker: "pedestrian_and_bicycle_tracker"
+    motorcycle_tracker: "pedestrian_and_bicycle_tracker"
+
+    # default tracker node parameters
+    publish_rate: 10.0
+    world_frame_id: map
+    enable_delay_compensation: false
```

Comment:
[Q]
Shouldn't the default value of enable_delay_compensation be set to true?

Reply:
For our vehicle, the answer is yes.